cmake/custom/main.cmake

```diff
@@ -35,16 +41,24 @@ add_custom_command(
 add_custom_target(
   binary-info
   ALL
+  COMMAND
+    touch ${PROJECT_SOURCE_DIR}/version.h.in
```

Comment:
Just to make sure we're platform-independent:
```
${CMAKE_BINARY_DIR} -E touch_nocreate ${PROJECT_SOURCE_DIR}/version.h.in
```

Reply:
I get this error
```
stig@pilates :: src/mrcpp/build-git ‹regen-git-hash*› » make                            148 ↵
-- Setting option BUILD_STATIC_LIBS: False
-- Using Eigen3: /usr (version 3.3.7)
-- Found packages:
 * OpenMP

-- Configuring done
-- Generating done
-- Build files have been written to: /home/stig/src/mrcpp/build-git
[  1%] Generating include/MRCPP/version.h
-- Git branch: regen-git-hash
-- Git last commit hash: 8eba1a069a50406cd55f-dirty
-- Git last commit author: Stig Rune Jensen
-- Git last commit date: Thu Dec 12 18:31:42 2019 +0100
make[2]: execvp: ./.: Permission denied
make[2]: *** [CMakeFiles/binary-info.dir/build.make:57: CMakeFiles/binary-info] Error 127
make[1]: *** [CMakeFiles/Makefile2:361: CMakeFiles/binary-info.dir/all] Error 2
make: *** [Makefile:141: all] Error 2
```

Reply:
If you do `touch` rather than `touch_nocreate`?

Reply:
same

Reply:
OK, I don't understand. Let's keep the original form given that it works.